src/python/Makefile

```diff
@@ -17,7 +17,7 @@ help:
 	@echo "create-env - create environment file if it doesn't exist (copy foglamp-env.example.yaml to foglamp-env.yaml)"
 	@echo "test - run all (src/python/tests and docs/check_sphinx) tests"
 	@echo "py-test - run src/python/tests"
```

Comment:
I know this is unrelated but build.sh uses --pytest .. I guess build.sh should be changed to use --py-test